tests/SideBySide.New/BulkLoaderAsync.cs

```diff
@@ -0,0 +1,377 @@
+﻿using System;
+using System.IO;
+using System.Linq;
+using System.Threading.Tasks;
+using MySql.Data.MySqlClient;
+using Xunit;
+using Dapper;
+
+namespace SideBySide
+{
+	[Collection("BulkLoaderCollection")]
+	public class BulkLoaderAsync : IClassFixture<DatabaseFixture>
+	{
+		public BulkLoaderAsync(DatabaseFixture database)
+		{
+			m_database = database;
+			//xUnit runs tests in different classes in parallel, so use different table names for the different test classes
+			string testClient;
+#if BASELINE
+			testClient = "Baseline";
+#else
+			testClient = "New";
+#endif
+			m_testTable = "test.BulkLoaderAsyncTest" + testClient;
+
+			var initializeTable = @"
+				create schema if not exists test;
```

Comment:
Right now we only use the Schema from the connection string in tests.  This is so that folks can do local testing against an existing MySQL database and not have random schemas appear/disappear when tests are run.  It also eliminates the need for the MySQL user to have SCHEMA level privileges.  Could this test be adapted to use the schema in the connection string like the others?